qutip/tests/test_gates.py

```diff
@@ -33,14 +33,17 @@
 
 import itertools
 import numpy as np
-from numpy.testing import (assert_, assert_allclose, assert_array_equal, 
+from numpy.testing import (assert_, assert_allclose, assert_array_equal,
                            run_module_suite)
 from qutip.states import basis, ket2dm
 from qutip.operators import identity, qeye, sigmax, sigmay, sigmaz
-from qutip.qip.operations.gates import (rx, ry, rz, phasegate, qrot, cnot, swap, iswap,
-                       sqrtswap, molmer_sorensen,
-                       toffoli, fredkin, gate_expand_3toN, 
-                       qubit_clifford_group, expand_operator)
+from qutip.qip.operations.gates import (rx, ry, rz, x_gate, y_gate, z_gate,
+                                        s_gate, t_gate, cy_gate, cz_gate,
+                                        cs_gate, ct_gate, phasegate, qrot,
+                                        cnot, swap, iswap, sqrtswap,
+                                        molmer_sorensen, toffoli, fredkin,
+                                        gate_expand_3toN, qubit_clifford_group,
+                                        expand_operator)
```

Comment:
While this is technically pep8-compliant, I think the spirit of the rules would suggest that
```python
from qutip.qip.operations.gates import (
    rx, ry, rz, ...,
    ...,
)
```
would make more sense (since it gives us much more space).

Reply:
Fixed on ad74aeb